### ISSUE TITLE
`migrate`

### ISSUE THREAD

**Issue by johnkors** [giggio/FluentMigrator.Runner.DNX#19]
Isn't the keyword to use this CLI extension clashing with dotnet CLI's own `migrate` , which migrates from `project.json` to new `msbuild / csproj`? What happens when using it with the latest CLIs which will have that keyword?

 [Documentation for `dotnet migrate`](https://docs.microsoft.com/en-us/dotnet/articles/core/preview3/tools/dotnet-migrate)

Maybe make it similar to how EF does it? `dotnet ef migrate` ..?  So for instance: `dotnet fluent migrate` ?

**Comment by giggio** [comment]
Yes, I know they are clashing. Hopefully people from FluentMigrator will get theirs ready in time for .NET Standard 2.0 and we can retire this one. They are targeting the work here: https://github.com/schambers/fluentmigrator/issues/712
Until then, we could evolve it. I haven't touch this code for a while, but a PR is sent I will check it out.
